packages/components-library/tests/unit/CheckboxFilter.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { createLocalVue, mount } from '@vue/test-utils'
+import { BootstrapVue } from 'bootstrap-vue'
+import CheckboxFilter from '@/components/filters/CheckboxFilter.vue'
+
+const localVue = createLocalVue()
+localVue.use(BootstrapVue)
+const stubs = ['font-awesome-icon']
+const options = async () => {
+  return [
+    { value: 'foo', text: 'Foo' },
+    { value: 'bar', text: 'Bar' },
+    { value: 'baz', text: 'Baz' }
+  ]
+}
+
+const buildWrapper = (value?: string[], maxVisibleOptions?: number) => {
+  const wrapper = mount(CheckboxFilter, {
+    localVue,
+    stubs,
+    propsData: {
+      name: 'name',
+      label: 'label',
+      value,
+      options,
+      maxVisibleOptions
+    },
+    listeners: {
+      input: (newVal: any) => { 
+        wrapper.setProps({ value: newVal });
+    }
+  }
+  })
+  return wrapper
+} 
+
+describe('CheckboxFilter.vue', () => {
+  let wrapper:any
+
+  describe('render the component', () => {
+    beforeEach(() => {
+      wrapper = buildWrapper([])
+    })
+  
+    it('matches the snapshot', () => {
+      expect(wrapper.element).toMatchSnapshot()
+    })
+
+  })
```

Comment:
describe is opzich handig om testen te groeperen(https://jestjs.io/docs/en/api#describename-fn), maar als je een groep hebt van 1 test, dan is het naar mijn idee geen groep en kan die describe weg.

Reply:
@jvanveen updates test as discussed